lib/bot.js

```diff
@@ -32,6 +32,8 @@ class Bot {
     this.commandCharacters = options.commandCharacters || [];
     this.ircNickColor = options.ircNickColor !== false; // default to true
     this.channels = _.values(options.channelMapping);
+    this.disableJoinPartMsgs = options.disableJoinPartMsgs;
```

Comment:
Thanks for adding a configuration option! I think it would be better to default it to off though, as that's the current behavior. [slack-irc](https://github.com/ekmartin/slack-irc) uses an object for this, so you can choose to enable joins and leaves selectively:
![image](https://cloud.githubusercontent.com/assets/3536982/24472221/8c0d09bc-148a-11e7-8422-ab0efd134a58.png)

I'm not sure if that's really necessary though, a single option might be more than enough. Maybe just `ircStatusNotices: true`.